frontend/app/src/components/home/ChatSummary.svelte

```diff
@@ -145,7 +144,7 @@
         delOffset = -60;
     }
 
-    $: chat = normaliseChatSummary($now, chatSummary, $typersByContext);
+    $: chat = normaliseChatSummary(chatSummary, $typersByContext);
```

Comment:
this changes the behaviour. By passing in $now we ensure that the function runs when $now changes (even though its value is not being directly used). Not sure if this is essential, but I'm sure that was intentional.

Reply:
Ahh ok, it was complaining about it not being used.

Reply:
Fixed